src/common/find_parameters_fi.f90

```diff
@@ -332,4 +336,98 @@ subroutine lambert_grid(fio, varname, proj4, nx, ny, xdim, ydim, igtype, gparam,
 
   end subroutine lambert_grid
 
+  !> determine gparam for polar_stereographic grids
+  subroutine polar_stereographic_grid(fio, varname, proj4, nx, ny, xdim, ydim, igtype, gparam, stat)
+    !> open fimex file
+    TYPE(FimexIO), intent(inout) :: fio
+    !> proj4 string
+    character(len=*), intent(in) :: varname
+    !> proj4 string
+    character(len=*), intent(in) :: proj4
+    !> xaxis-size
+    integer, intent(in) :: nx
+    !> yaxis-name
+    integer, intent(in) :: ny
+    !> xaxis-name
+    character(len=*), intent(in) :: xdim
+    !> yaxis-name
+    character(len=*), intent(in) :: ydim
+    !> grid type
+    integer, intent(out) :: igtype
+    !> grid parameters
+    real, intent(out) :: gparam(6)
+    !> error code (0 for success)
+    integer, intent(out) :: stat
+
+    real(kind=real64) :: pi
+    real(kind=real64) :: lat0, incr, startX, startY
+    real(kind=real64), allocatable, target :: dims(:), latlons(:)
+    character(1024) :: pval, longname, latname
+    integer :: ndims
+
+    PI = 4.D0*DATAN(1.D0)
```

Comment:
real(kind=real64) :: pi = 4.d0*datan(1.d0)

Reply:
can be refactored when we make pi a general constant